apps/wallet-dashboard/lib/utils/analytics/amplitude.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import * as amplitude from '@amplitude/analytics-browser';
+import { LogLevel, TransportType, type UserSession } from '@amplitude/analytics-types';
+import { PersistableStorage } from '@iota/core';
+
+import { ampli } from './ampli';
+
+const IS_PROD_ENV = process.env.NODE_ENV == 'production';
+
+export const persistableStorage = new PersistableStorage<UserSession>();
+
+export async function initAmplitude() {
+    await ampli.load({
+        environment: 'iotawalletdashboard',
```

Comment:
In explorer and wallet, we did `environment: IS_PROD_ENV ? 'production' : 'development'`, why do we only have one environment here?
For the Sentry configuration of the dashboard, we also differentiate between prod and dev

Reply:
We are limited to 5 projects, so there is only one for the dashboard 🤐